src/kotlinSessions/SessionTwo.kt

```diff
@@ -55,10 +59,99 @@ fun main() {
     for (number in range step 1) {
         println("Before continue")
         if (number > 3) {
+            println("Inside the condition !")
             // skips the lines after it and continue looping
             continue
         }
         println("After continue")
     }
+
     //-----------------------------------------------------------------------
+
+
+    //--------------------------------- Exercises --------------------------------------------
+
+    // The factorial function (symbol: !) refers to multiplying all whole numbers from our chosen number down to 1.
+    // Rule = n! = n × (n−1) x (n-2) x (n-3) x .........
+    // see more at : https://www.mathsisfun.com/numbers/factorial.html
+    // request user input for a number value
+    val number: String? = readLine()
```

Comment:
I think this a little hard code

Reply:
MMm, i will try to simplify as far as i can !